packages/benchmark-utils/src/index.ts

```diff
@@ -0,0 +1,51 @@
+import fs from "fs";
+import {BENCH_DIR} from "./constant";
+import Benchmark from "benchmark";
+
+export interface BenchSuite {
+  testFunctions: Function[];
+  file: string;
+  setup?: Function;
+  teardown?: Function;
+}
+
+export const createReportDir = (): string => {
+  const curDate: string = new Date().toISOString();
+  const dir: string = BENCH_DIR + `${curDate}/`;
+
+  // If benchmark directory doesn't exist create it
+  if (!fs.existsSync(BENCH_DIR)) {
+    fs.mkdirSync(BENCH_DIR);
+  }
+
+  // Create the current benchmark folder
+  if (!fs.existsSync(dir)) {
+    fs.mkdirSync(dir);
+  }
+  return dir;
+};
+
+export const writeReport = (file: string, data: string) => {
+  console.log(data);
```

Comment:
remove

Reply:
But I actually wan't to see result in console as well :sweat_smile:

Reply:
Use a logger? Lol idk what's the output

Reply:
same thing that goes into file. We dont have logger available there nor there is a point in logger for dev tool haha

Reply:
oh ya true..